tests/phpunit/integration/includes/Fixture/DummyInterface.php

```diff
@@ -3,6 +3,5 @@
 namespace Google\Web_Stories\Tests\Integration\Fixture;
 
 interface DummyInterface {
-
-	public function get_dummy();
+	public function get_dummy(); // phpcs:ignore SlevomatCodingStandard.TypeHints.ReturnTypeHint.MissingNativeTypeHint
```

Comment:
Could just to `: void` here and in `DummyClassWithDependency`?

Reply:
No, the downstream method doesn't return void.